microsoft-edge/extensions-chromium/publish/contact-us.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Contact support information for Microsoft Edge extension development.
+title: Contact Edge extensions support
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/19/2021
+ms.topic: support
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, support
+---
+
+# Contact Edge extension support
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
+1. If you have any issues or feedback regarding your publishing experience using Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
+1. To feature your extension in a collection on the Edge Add-ons home page, submit your request using the [Submit a request to add an extension to the collections on the MS Edge Add-ons home page][CollectionsForm] form. 
```

Comment:
Add another section here:

If your extension is not working as expected with Microsoft Edge or you have a question regarding how to develop an extension, check out these links: Supported APIs for Microsoft Edge extensions and Port Chromium extensions to Microsoft Edge. For any other questions, write to us at xxxx.